tests/components/element-card/element-card.spec.tsx

```diff
@@ -0,0 +1,109 @@
+import { fireEvent, render, screen } from "@testing-library/react";
+import "jest-canvas-mock";
+import { ElementCard } from "src/components";
+import { EElementType, TElement } from "src/context-providers";
+import { ELEMENT_BUTTON_LABELS } from "src/data/elements-data";
+import { TestHelper, mockBuilderState } from "src/util/test-helper";
+
+describe("element-card", () => {
```

Comment:
We usually use the component's name. So here it should be `ElementCard`

Reply:
This has been changed in the PR for CCUBE-1218
![Uploading image.png…]()